src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -289,6 +289,78 @@ begin
   rw (congr_fun (congr_arg (@nhds E) hp.1) 0),
   exact add_group_filter_basis.nhds_zero_has_basis _,
 end
+
+/-- The `x`-neighbourhoods of a space whose topology is induced by a family of seminorms
+are exactly the sets which contain seminorm balls around `x`.-/
+lemma with_seminorms.mem_nhds_iff (hp : with_seminorms p) (x : E) (U : set E) :
+  U ∈ nhds x ↔ ∃ (s : finset ι) (r > 0), (s.sup p).ball x r ⊆ U :=
+begin
+  rw [hp.with_seminorms_eq, (p.add_group_filter_basis.nhds_has_basis x).mem_iff' U],
+  split,
+  { rintros ⟨V, V_basis, hVU⟩,
+    rcases p.basis_sets_iff.mp V_basis with ⟨i_set, r, r_pos, V_ball⟩,
```

Comment:
```suggestion
    rcases p.basis_sets_iff.mp V_basis with ⟨_, _, r_pos, V_ball⟩,
```
I was about to suggest `i_set` to `s`, but since it is not used, we can get rid of it altogether.